### DISCUSSION TITLE
Template for mobile auth ?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [mockoon/mockoon#1140] [Help]
I plan to build a flutter app and uee mockoon for the api development.

Sadly nothing much heve comeup in regards of mockoon eirh an example flutter app doing auth.

Any examples templates repros i could study would love to learn how to best approach this🙏

**Comment by 255kb** [comment]
There is currently no automated auth mechanism in Mockoon. You will have to create your endpoints manually, for example a `POST /login` endpoint that returns a JSON object containing a token. 
You can also make use of response rules to verify if a username and password are present in the body, and return a 200 or a 401 if not. 
I suggest that you take a look at this tutorial:
https://mockoon.com/tutorials/responses-and-rules/
